common/metrics/src/lib.rs

```diff
@@ -310,30 +303,33 @@ impl Http {
     }
 
     pub fn add_request_success(&self, start: Instant) {
+        let duration = start.elapsed();
```

Comment:
Before, we would emit slightly different timestamp values since each `observe` call computed its own timestamp. Now we use the same duration for both metrics so they are consistent with each other.